ui/client/components/table/DeployedFilter.tsx

```diff
@@ -0,0 +1,25 @@
+import TableSelect from "./TableSelect"
```

Comment:
Please change `DeployedFilter` to `StatusFilter`. Right now we filter only by deployed / not deployed - but in future who knows ;)

Reply:
[6]